Update README.md

krishna95 opened the pull request.

krishna95 committed:
Update README.md

krishna95 commented:
@hargup

hargup commented:
Create a new section for research papers.

krishna95 committed:
Update README.md

krishna95 commented:
@hargup

hargup commented on README.md:
```diff
@@ -17,9 +17,19 @@ University Courses and MOOCS
   Richard Szeliski
+
+Research Papers
+
```
Remove this new line to make "Research Papers" a heading, you can see the rendered text from a button on the top of this diff.

hargup commented:
And a better commit message please.

krishna95 committed:
Updated README.md 

README.md updated according to the changes suggested. Research Papers has been made an heading and a separate columns of the papers has been made

hargup commented:
Thanks for the changes.

krishna95 commented:
I should rather thank you to teach me to do each task perfectly and not
take anything lightly.

On Wed Dec 03 2014 at 11:21:17 PM Harsh Gupta notifications@github.com
wrote:

> Thanks for the changes.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/AGV-IIT-KGP/awesome-computer-vision/pull/2#issuecomment-65454920
> .

hargup commented:
> I should rather thank you to teach me to do each task perfectly and not
> take anything lightly.

Well I did, I think even your new commit message wasn't good enough. And we definitely don't need three commits for this small change, but I didn't ask you squash the commits into one. I'll be more stringent from next time :wink: .

krishna95 commented:
Sure :)

On Wed Dec 03 2014 at 11:35:44 PM Harsh Gupta notifications@github.com
wrote:

> I should rather thank you to teach me to do each task perfectly and not
> take anything lightly.
> 
> Well I did, I think even your new commit message wasn't good enough. And
> we definitely don't need three commits for this small change, but I didn't
> ask you squash the commits into one. I'll be more stringent from next time [image:
> :wink:] .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/AGV-IIT-KGP/awesome-computer-vision/pull/2#issuecomment-65457315
> .
